code/modules/mob/living/simple_animal/distortion/legend/papa_bongy.dm

```diff
@@ -0,0 +1,332 @@
+	name = "Papa Bongy"
+	desc = "A human with the head of a chicken and deformed arms. It appears to be carrying a sack of raw chickens."
+	icon = 'ModularTegustation/Teguicons/64x64.dmi'
+	icon_state = "papa_bongy"
+	maxHealth = 2500
+	health = 2500
+	pixel_x = -16
+	base_pixel_x = -16
+	fear_level = HE_LEVEL
+	move_to_delay = 5
+	damage_coeff = list(RED_DAMAGE = 0.8, WHITE_DAMAGE = 2, BLACK_DAMAGE = 1.2, PALE_DAMAGE = 1.5)
+	melee_damage_lower = 15
+	melee_damage_upper = 20
+	melee_damage_type = WHITE_DAMAGE
+	stat_attack = HARD_CRIT
+	attack_sound = 'sound/distortions/papa_bongy/PapaBongy_Coin1.ogg'
+	attack_verb_continuous = "smacks"
+	attack_verb_simple = "smack"
+	ranged = TRUE
+	ranged_cooldown_time = 10 SECONDS
+	del_on_death = FALSE
+	faction = list("bongy")
+
+	ego_list = list(
+		/obj/item/ego_weapon/lance/lifestew_lance,
+		/obj/item/clothing/suit/armor/ego_gear/he/lifestew
+		)
+	//The egoist's name, if specified. Otherwise picks a random name.
+	egoist_names = list("Hong Gil-Dong")//This is a korean John Doe, as the Eunbong owner is unnamed.
+	gender = MALE
+	egoist_attributes = 40
+	egoist_outfit = /datum/outfit/job/civilian
+	loot = list(/obj/item/documents/ncorporation)
+
+	var/can_act = TRUE
+	var/ding_delay = 1.5 SECONDS
+	var/throw_delay = 2 SECONDS
+	var/unit_list = list()
+	var/units_max = 8
+	var/happy = FALSE
+	var/yucky_list = list(
+	/datum/reagent/consumable/enzyme,
+	/datum/reagent/consumable/soymilk,
+	/datum/reagent/consumable/astrotame,
+	)
+	var/bad_pizza_list = list(
+	/obj/item/food/pizzaslice/donkpocket,
+	/obj/item/food/pizzaslice/dank,
+	/obj/item/food/pizzaslice/pineapple,
+	/obj/item/food/pizzaslice/moldy,
+	)
+
+/mob/living/simple_animal/hostile/distortion/papa_bongy/Initialize(mapload)
+	..()
+	var/units_to_add = list(
+		/mob/living/simple_animal/hostile/bongy_hostile = 1,
+		)
+	AddComponent(/datum/component/ai_leadership, units_to_add, 8, TRUE, TRUE)
+
+/mob/living/simple_animal/hostile/distortion/papa_bongy/death(gibbed)
+	density = FALSE
+	animate(src, alpha = 0, time = 10 SECONDS)
+	QDEL_IN(src, 10 SECONDS)
+	..()
+
+//Unmanifesting is not linked to any proc by default, if you want it to happen during gameplay, it must be called manually.
+/mob/living/simple_animal/hostile/distortion/papa_bongy/attacked_by(obj/item/I, mob/living/user)
+	. = ..()
+	if(istype(I, /obj/item/food/fried_chicken))//Fried chicken will require certain condiments to de-manifest in the future. For now, keep it simple
+		for(var/goop in I.reagents.reagent_list)
+			var/datum/reagent/R = goop
+			if(istype(R, /datum/reagent/consumable/capsaicin))
+				if(R.volume >= 40)//is it REALLY spicy?
+					say("BUGAAAAAAAAAAAAAAAAAAAAAAAAWK!")
+					to_chat(user, "[src] grimaces, sputters and coughs! It begins thrashing about, and you feel the temperature around you rise...")
+					new /mob/living/simple_animal/hostile/distortion/papa_bongy/spicy(get_turf(src))
+					qdel(src)
+					break
```

Comment:
This likely works on the spicy papa bongy, since it's a child of papa_bongy. This means that this can be done recursively. Forever. That should probably be fixed.